src/index.html

```diff
@@ -17,11 +18,360 @@
+        </h1>
+        <p class="card__code">Product code: 195434</p>
+        <!-- #region stars -->
+
+        <div class="card__reviews-block">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <!-- #endregion -->
+        <div class="card__value">
+          <p class="card__price">Price:</p>
+          <p class="card__price-number">$2,199</p>
+        </div>
+        <button class="card__button">buy</button>
+      </div>
+      <!-- #endregion -->
+      <!-- #region card 6-->
+      <div class="card">
+        <img
+          class="card__image"
+          src="../src/images/imac.jpeg"
+          alt="imac"
+        />
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__code">Product code: 195434</p>
+        <!-- #region stars -->
+
+        <div class="card__reviews-block">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <!-- #endregion -->
+        <div class="card__value">
+          <p class="card__price">Price:</p>
+          <p class="card__price-number">$2,199</p>
+        </div>
+        <button class="card__button">buy</button>
+      </div>
+      <!-- #endregion -->
+      <!-- #region card 7-->
+      <div class="card">
+        <img
+          class="card__image"
+          src="../src/images/imac.jpeg"
+          alt="imac"
+        />
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__code">Product code: 195434</p>
+        <!-- #region stars -->
+
+        <div class="card__reviews-block">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <!-- #endregion -->
+        <div class="card__value">
+          <p class="card__price">Price:</p>
+          <p class="card__price-number">$2,199</p>
+        </div>
+        <button class="card__button">buy</button>
+      </div>
+      <!-- #endregion -->
+      <!-- #region card 8-->
+      <div class="card">
+        <img
+          class="card__image"
+          src="../src/images/imac.jpeg"
```

Comment:
The image source path `../src/images/imac.jpeg` might not be correct depending on your file structure. Ensure the path is correct relative to the HTML file location.